src/platforms/javascript/common/install/lazy-load-sentry.mdx

```diff
@@ -22,39 +26,77 @@ The loader is less than 1kB gzipped and includes the `Sentry.init` call with y
+Finally, if you want to control the timing yourself, you can call `Sentry.forceLoad()`. You can do this as early as immediately after the loader runs (which has the same effect as setting `data-lazy="no"`) and as late as the first unhandled error, unhandled promise rejection, or call to `Sentry.captureMessage` or `Sentry.captureEvent` (which has the same effect as not calling it at all). Note that you can't delay loading past one of the aforementioned triggering events.
 
-Go into the Sentry web UI, view _Settings -> Projects -> Client Keys (DSN)_, then press the **Configure** button. From here, you can view the options to configure your DSN and select which SDK version the Loader should load.
+## SDK Configuration
 
-<Note>
+The loader script includes a call to `Sentry.init` with one configuration option: your DSN. If you want to [configure your SDK](../../configuration/options/) beyond that, you'll need a second script tag, in which you'll call `Sentry.onLoad`. This script must come _after_ the main loader script.
 
-It can take a few minutes until the change is visible in the code, since it’s cached.
-
-</Note>
+```html
+<script
+  src="https://js.sentry-cdn.com/___PUBLIC_KEY___.min.js"
+  crossorigin="anonymous"
+></script>
+<script>
+  Sentry.onLoad(function() { ... });
+</script>
+```
 
-![JavaScript Loader Settings](js-loader-settings.png)
+`Sentry.onLoad` is a function that only the loader provides, and - as the name suggests - it sets a function to be run once the full SDK has been loaded. In that function, you can configure your SDK exactly the way you would were you using the CDN, with one difference: your `Sentry.init` call doesn't need to include your DSN, since it's already been set.
 
-## **Current limitations**
+```html
+<script>
+  Sentry.onLoad(function() {
+    Sentry.init({
+      release: " ... ",
+      environment: " ... "
+    });
+    Sentry.configureScope(scope => {
+      scope.setTag( ... );
+    });
+    // etc.
+  });
+</script>
+```
 
-Because we inject our SDK asynchronously, we will only monitor *global errors* and *unhandled promise* for you until the SDK is fully loaded. That means that we might miss breadcrumbs during the download.
+## **Limitations**
 
-For example, a user clicking on a button on your website is making an XHR request. We will not miss any errors, only breadcrumbs and only up until the SDK is fully loaded. You can reduce this time by manually calling `forceLoad` or set `data-lazy="no"`.
+Because the loader injects the SDK asynchronously, only *unhandled errors* and *unhandled promise rejections* will be caught and buffered before the SDK is fully loaded. Specifically, neither our [performance features](../../performance/) nor the capturing of any [breadcrumb data](../../enriching-events/breadcrumbs/) will be available until the SDK is fully loaded and initialized. To reduce the amount of time these features are unavailable, set `data-lazy="no"` or call `forceLoad()` as described above.
```

Comment:
```suggestion
Because the loader injects the SDK asynchronously, only *unhandled errors* and *unhandled promise rejections* will be caught and buffered before the SDK is fully loaded. Specifically, neither our [performance features](/product/performance/) nor the capturing of any [breadcrumb data](../../enriching-events/breadcrumbs/) will be available until the SDK is fully loaded and initialized. To reduce the amount of time these features are unavailable, set `data-lazy="no"` or call `forceLoad()` as described above.
```

Reply:
Because Cordova doesn't support performance, the common content is failing on this link. Switching to product.